app/main.py

```diff
@@ -1,25 +1,92 @@
 from abc import ABC
+from typing import Type
```

Comment:
The `IntegerRange` class is defined but not used anywhere in the code. If it's intended to be used as a descriptor for the `Visitor` class attributes, you should implement it accordingly. Otherwise, consider removing it if it's not needed.